core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -586,14 +586,14 @@ function (array $matches) use ($arrData)
 
 		$evaluateExpression = function ($strExpression) use ($arrData)
 		{
-			if (!preg_match('/^([^=!<>\s]+)([=!<>]+)(.+)$/s', $strExpression, $arrMatches))
+			if (!preg_match('/^([^=!<>\s]+)[ ]*([=!<>]+)(.+)$/s', $strExpression, $arrMatches))
```

Comment:
Can be simplified to ` *` or even ` ?` (the latter would allow not more than one space).

Reply:
I’d go with `) *(`